Dockerfile

```
@@ -43,14 +43,14 @@ ARG DEV_MODE="false"
 ENV DEV_MODE=${DEV_MODE}
 ENV NODEROOT=/home/aeternity/node/local
 ARG NODE_VERSION=7.0.0
-ARG NODE_URL=https://github.com/aeternity/aeternity/releases/download/v${NODE_VERSION}/aeternity-v${NODE_VERSION}-ubuntu-x86_64.tar.gz
 ENV NODEDIR=/home/aeternity/node/local/rel/aeternity
 RUN mkdir -p ./local/rel/aeternity/data/mnesia
-RUN curl -L --output aeternity.tar.gz ${NODE_URL} && tar -C ./local/rel/aeternity -xf aeternity.tar.gz
 
+COPY --from=aeternity/aeternity:v7.0.0-bundle /home/aeternity/node ./local/rel/aeternity
```

- you shouldn't need the `-bundle` flavour
- the version argument should be used to allow rebuilds with different version without changing the dockerfile and to not repeat the version below

```suggestion
COPY --from=aeternity/aeternity:v${NODE_VERSION} /home/aeternity/node ./local/rel/aeternity
```